tools/Makefile.mk

```diff
@@ -1,5 +1,6 @@
-include tools/ola_trigger/Makefile.mk
+include tools/ja-rule/Makefile.mk
```

Comment:
This needs to be excluded on WIN32 as the StdIn handler it uses doesn't work for E1.33 so won't here either. It should also only be built if libusb is present.